src/Tags/Concerns/QueriesOrderBys.php

```diff
@@ -30,6 +30,10 @@ protected function parseOrderBys()
 
         $piped = Arr::getFirst($this->params, ['order_by', 'sort'], $this->defaultOrderBy());
 
+        if ($piped === null) {
```

Comment:
when would this ever be null? The `defaultOrderBy` for tags returns a string.

Reply:
I was catching nulls for the default during my testing:

https://github.com/statamic/cms/blob/3.3/src/Tags/Concerns/QueriesOrderBys.php#L43-L46

Reply:
so...don't return null there?

Reply:
Ha! Could also do that! I've made that change, and it doesn't seem to break anything